javascript/login.js

```diff
@@ -0,0 +1,32 @@
+// auth.onAuthStateChanged(firebaseUser => { });
+var config = {
+    apiKey: "AIzaSyChvgSJwWRcfCZ1m9onBBb3Zyf3dZ5bRYQ",
+    authDomain: "walkme-app-1520459500880.firebaseapp.com",
+    databaseURL: "https://walkme-app-1520459500880.firebaseio.com",
+    projectId: "walkme-app-1520459500880",
+    storageBucket: "",
+    messagingSenderId: "829716536506"
+  };
+  firebase.initializeApp(config);
+  //GRABBING ELEMENTS FROM DOM
+  var txtEmail = document.getElementById('txtEmail');
+  var txtPassword = document.getElementById('txtPassword');
+  var btnLogin = document.getElementById('btnLogin');
+  var btnSignUp = document.getElementById('btnSignUp');
+  var btnLogout = document.getElementById('btnLogout');
+  
+  //ADD LOGIN EVENT
+  
+  btnLogin.addEventListener('click', e => {
+    // get email and password
+    var email = txtEmail.value;
+    var pass = txtPassword.value;
+    // sign in
+    var firebaseUser = firebase.database().ref('users/00001');
```

Comment:
These text boxes scare me a bit.  This would mean when people enter their passwords, we could theoretically gather user names and passwords with a line of code.  The login function works because when I did the jimmyjohn@gmail.com (using the password you left on the database) the console came back with true.

I'm willing to commit it for now, but highly suggest none of us test this out with our real info for awhile!